internal/locate/region_cache.go

```diff
@@ -2646,6 +2646,13 @@ func (s *StoreHealthStatus) markAlreadySlow() {
 	s.updateSlowFlag()
 }
 
+// resetSlowScore resets the slow score to 0. It's used for test.
+func (s *StoreHealthStatus) resetSlowScore() {
+	s.clientSideSlowScore.resetSlowScore()
+	s.updateTiKVServerSideSlowScore(0, time.Now())
```

Comment:
Can this be done in any other way? I think it might break the consistency of their internal states.
By the way, the minimal valid value (for the slow scores in both client side and tikv side) should be 1 instead of 0.

Reply:
Maybe you can consider reset the whole `healthStatus` field in `Store` struct, if it won't cause races. Another choice is to fake `tick` calls repeatedly until the two scores decay to 1.